.probo.yaml

```diff
@@ -4,5 +4,7 @@ steps:
     makeFile: build-bear.make
     profileName: bear
     runInstall: true
+  - name: Install bear_skin dependencies
```

Comment:
I wasn't sure at this point actually and said bear_skin thinking it was required for both bear_coat and bear_skin. After checking with Alban it's actually not for bear_skin so this is wrong. The packaging system blocking our libraries is blocking the building of the bear release properly on D.O. We may need to just live without bear_coat as the default and leave it bear_skin to avoid this issue. This code wouldn't be here if the semantic UI was on the packaging whitelist...